tensorflow/tools/benchmark/benchmark_model.cc

```diff
@@ -272,7 +272,11 @@ Status TimeMultipleRuns(double sleep_seconds, int num_runs,
     // This can be helpful to determine the effect of mobile processor
     // scaling and thermal throttling.
     if (sleep_seconds > 0.0) {
+#ifdef _WIN32
```

Comment:
Minor nit, but can you add an include to [tensorflow/core/platform/platform.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/platform/platform.h) and use the PLATFORM_WINDOWS macro instead? That is a bit more idiomatic.